llm/llama.cpp/generate_darwin_amd64.go

```diff
@@ -9,8 +9,10 @@ package llm
 //go:generate git -C ggml apply ../patches/0004-metal-add-missing-barriers-for-mul-mat-2699.patch
 //go:generate cmake -S ggml -B ggml/build/cpu -DLLAMA_ACCELERATE=on -DLLAMA_K_QUANTS=on -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_OSX_ARCHITECTURES=x86_64 -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0
 //go:generate cmake --build ggml/build/cpu --target server --config Release
+//go:generate mv ggml/build/cpu/bin/server ggml/build/cpu/bin/ollama-runner
```

Comment:
There's no parameter in the server's CMakeLists.txt to set to name of the output executable, so renaming it manually. I prefer this to patching the CMakeLists.txt

Reply:
This is unnecessary since this is the input file to the embedded file system. Instead you can change the destination file name when copying the embedded file

Reply:
I prefer this way since we can keep the destination file logic simple, just copying whatever files are in the embedded directory.